Web/Marketplace/src/components/loginPage/LoginPage.tsx

```diff
@@ -18,27 +18,34 @@ function LoginPage() {
         }).then(()=> console.log("LogIn", logInnData))
     }
     return (
-        <div>
-            <div className={"mainBox"}>
-                <p>Login</p>
-                <label>Email</label>
+        
+            <div className={"mainLoginBox"}>
+                <form>
+                
+                <h1>Login</h1>
                 <input
+                    placeholder="Type your mail ..."
+                    id="mail"
+                    className="mailLoginInput"
                     type="text"
                     required
                     onChange={(e) => logInnData.email = e.target.value}
                 />
 
-                <label>Password</label>
+                
                 <input
-                    type="text"
+                    placeholder="Type your password ..."
+                    id="password"
+                    className="passwordLoginInput"
+                    type="password"
                     required
                     onChange={(e) => logInnData.password = e.target.value}
                 />
-                <button onClick={HandleLogIn}>
-                    LogIn
+                <button className="loginButton" onChange={HandleLogIn}>
```

Comment:
Why is there an onChange event in the button? I think it should be onClick instead.